### PR TITLE
Ri doc

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: RI.md [hunk 1]
```diff
@@ -0,0 +1,1258 @@
...
+
+Example commands for printing Nokogiri documents
+(assuming that the gem is installed);
+see {Gem Documents}[rdoc-ref:RI.md@Gem+Documents]:
+
+| Command                               | Prints Document For                                 |
+|---------------------------------------|-----------------------------------------------------|
+| ri Nokogiri::HTML4::Document          | Class Nokogiri::HTML4::Document.                    |
+| ri Nokogiri                           | Module Nokogiri.                                    |
+| ri Nokogiri::HTML4::Document::parse   | 	Class method Nokogiri::HTML4::Document::parse.     |
```
Comment (olleolleolle) [inline RI.md:37]: Perhaps the space was unintentional?

```suggestion
| ri Nokogiri::HTML4::Document::parse   | Class method Nokogiri::HTML4::Document::parse.     |
```

File: RI.md [hunk 2]
```diff
@@ -0,0 +1,1258 @@
...
+**Note**: Throughout this page, the term _prints_
+means <i>puts onto `$stdout`</i>, not <i>sends to a print device</i>.
+
+Example commands for printing Ruby documents;
+see {Ruby Documents}[rdoc-ref:RI.md@Ruby+Documents]:
+
+| Command             | Prints Document For            |
+|---------------------|--------------------------------|
+| ri IO               | Class IO.                      |
+| ri Enumerable       | Module Enumerable              |
```
Comment (olleolleolle) [inline RI.md:23]: Perhaps a period after each?

```suggestion
| ri Enumerable       | Module Enumerable.             |
```

File: RI.md [hunk 3]
```diff
@@ -0,0 +1,1258 @@
...
+    $ ri
+    Enter the method name you want to look up.
+    You can use tab to autocomplete.
+    Enter a blank line to exit.
+    >> File
+    >> IO#readlines
+    >> ruby:
+    ```
+
+These tables summarizes `ri` _name_ values:
```
Comment (olleolleolle) [inline RI.md:142]: Minor grammar:

```suggestion
These tables summarize `ri` _name_ values:
```

**Comment by olleolleolle** [comment]
I interjected comments, they are to be read as encouragement and cheering from the sidelines!

**Comment by BurdetteLamar** [comment]
Thanks, @olleolleolle.  I did not mean to open a PR just yet (still a long way to go), so closing.

**Pull request closed without merging**